system/integrations/pgp/index.rst

```diff
@@ -0,0 +1,111 @@
+PGP
+======
+
+
+Introduction
+------------
+
+PGP is another method for secure email communication.
+With PGP, you can exchange **signed** and **encrypted** messages with others.
+
+Signing
+   is a proof that a message hasn’t been manipulated on it's way.
+
+   In other words, it guarantees a message’s **integrity** and **authenticity**.
+
+Encryption
+   scrambles a message so that it can only be unscrambled by the intended recipient.
+
+   In other words, it guarantees **privacy** and **data security**.
+
+.. figure:: /images/system/integrations/pgp/pgp_ticket_creation.png
+   :alt: Screenshot of ticket creation with encrypt and sign buttons
+   :align: center
+
+   Once PGP has been enabled, 🔒 **Encrypt** and ✅ **Sign** buttons will appear in the ticket composer.
+
+.. note:: **Sign button not visible?**
+
+   Please note that the signing of emails is based on the outgoing email account. 
+   That means you have to choose a group with a sender email account, which has a private key assigned.
+
+Handling of keys
+-----------------
+To use the pgp function, you fist have to enable the integration (PGP). 
+When you swith the toggle to *enabled*, you can add keys by clicking the *add key* button. 
+The keys can be imported from a file or you can paste the content of the key in the text box.
+
+.. note:: **Which keys do I have to import?**
+
+   For **signing** *outgoing* emails, you have to import the private key of your sender email account.
+   
+   For **encrypting** *outgoing* emails, you have to import the public key of the recepient's email account.
+
+   For **checking the signature** of signed *incoming* emails, you have to import the public key of the sender.
+
+   For **decrypting** of encrypted *incoming* emails, you have to import the public key of the sender. 
```

Comment:
Hm, not exactly: **private** key of the system sender (in this case "recipient", since it's incoming) must be present.